src/App.svelte

```diff
@@ -17,6 +17,8 @@
         isLoggedIn = status;
     });
 
+    const ws = new WebSocket(`wss://${location.host}/subscribe`);
```

Comment:
Why is this here?

Reply:
I might have pasted a other code, sorry for that